### PR TITLE
Don't block the main thread when cloudifying a project so the UI updates

### PR DESCRIPTION
**Pull request by suricactus**
@nirvn can you confirm it works fine? I tested with two (small) projects and seems to be much better UX.

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-1672 Don't block the main thread when cloudifying a project](https://app.clickup.com/t/2192114/QF-1672)

File: qfieldsync/gui/cloud_create_project_widget.py [hunk 1]
```diff
@@ -168,16 +171,19 @@ def cloudify_project(self):
...
+                critical_message = self.tr(
+                    "The project could not be converted into the export directory."
+                )
+                self.iface.messageBar().pushMessage(critical_message, Qgis.Critical, 0)
+                self.close()
+                return
+
+        self.timer.singleShot(0, convert)
 
         self.create_cloud_project()
```
Comment (m-kuhn) [inline qfieldsync/gui/cloud_create_project_widget.py:188]: Is it a problem that line 188 is now done before create is executed?

Comment (suricactus) [reply qfieldsync/gui/cloud_create_project_widget.py:188]: If you have a fast internet - no :) Thanks for spotting it.

**Review by nirvn — APPROVED** [review]
Logic is now sound ;)

**Pull request merged**